scalarm_client_csharp/Client.cs

```diff
@@ -26,6 +27,42 @@ public Client(string baseUrl)
             // this.AddHandler("application/json", new JsonConvertDeserializer());
 		}
 
+		//absolutePath must be to file with extension .tar.gz
+		public string GetSimulationScenarioFiles(string scenarioId, string absolutePath)
```

Comment:
@Andriu01 - nie zwróciłem na to uwagi wcześniej - ale tą metodę można spokojnie przenieść do SimulationScenario. Albo coś źle powiedziałem wcześniej (?)